### PR TITLE
Try move auth

### PR DESCRIPTION
**Pull request by theRealImy**
This is the PR for the move auth from solid-ui to solid-logic.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,26 +32,27 @@
...
-    "solid-ui": "^2.4.15",
+    "solid-logic": "1.3.13-60e05a84",
+    "solid-ui": "2.4.16-74e478b4",
     "validate-color": "^2.2.1"
   },
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.0",
     "@babel/preset-typescript": "^7.16.0",
+    "@nestjs/cli": "^8.2.0",
```
Comment (angelo-v) [inline package.json:47]: **question** what are we using that for?

File: src/integration-tests/profile-card.spec.ts [hunk 2]
```diff
@@ -35,9 +34,10 @@ describe("profile-pane", () => {
           ];
       .
   `;
-      parse(turtle, store, doc.uri);
+      parse(turtle, solidLogicSingleton.store, doc.uri);
       result = pane.render(subject, context);
     });
+    afterAll(() => { solidLogicSingleton.store.removeDocument(doc)})
```
Comment (angelo-v) [inline src/integration-tests/profile-card.spec.ts:40]: **praise** ah yes, thats better :)

**Review by angelo-v — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (theRealImy) [reply package.json:47]: It used to be pulled in by some other lib and it is not anymore... related to webpack: https://stackoverflow.com/questions/66809411/webpack-typescript-errors-on-fresh-nestjs-app

Comment (angelo-v) [reply package.json:47]: That's weird, since we are not using NestJS at all, I think this should be investigated further

Comment (theRealImy) [reply package.json:47]: I will resolve this conversation for now to be able to move towards closing the PR. We should take a close look at all our dependencies of all modules from time to time, agree.

**Pull request merged**